src/src5 implementation.md

```diff
@@ -0,0 +1,113 @@
+## SRC-5: A comprhensive Overview
+
+SRC-5 is a standard for smart contract interface introspection in Starknet, inspired by the Ethereum ERC-165 standard. It provides a method for contracts to publish and detect the interfaces they implement, ensuring standardized interaction. It provides a method for contracts to publish and detect the interfaces they implement, ensuring standardized interaction.  You can find more information and details refer to the [SRC-5 specification](https://github.com/starknet-io/SNIPs/blob/main/SNIPS/snip-5.md#simple-summary) and the [Ethereum ERC-165 standard](https://eips.ethereum.org/EIPS/eip-165).
```

Comment:
Remove sentence duplications, add some breaklines, fix multiple whitespaces, move link for ERC-165 directly when you mention it first.